### PR TITLE
Add option to accept id attributes from <path>

### PR DESCRIPTION
**Pull request by lobenichou**
### The PR
Currently, the code ignores any id attributes. This PR handles id attributes on `<path>` and returns the id as a feature property.  It also comments out the svgo plugin `cleanupIDs` because it stripped attributes altogether. 

### Reasoning 
The reasoning behind this change is that we want to be able to edit the SVGs in  Mapbox Studio and possibly do some data-join at runtime. Without a specific property, this is hard to accomplish. 

We could possibly handle multiple attributes on `<path>` but for now, this will make it easier to do data-driven styling using the svg in Studio. 

### To test

Use the `WSJ.svg` from this link: http://bl.ocks.org/veltman/c7bfb3d4a3817f7ee0bf2dd19ff058c1. 

Upload it, and download it. Check to make sure the GeoJSON has an `id` property. If no `id` is passed, the properties will be empty.

### CODE REVIEW

**Comment by tristen** [comment]
@lobenichou thanks for the PR! So I'm trying that `wsj.svg` file and its returning blank properyt ID's

screen shot 2018-11-06 at 1 18 37 pm

Can we also omit `properties.id` if one doesn't exist? instead of `id: ""`

File: src/components/app.js [hunk 1]
```diff
@@ -50,7 +50,6 @@ let App = class App extends React.PureComponent {
     ], {
       type: 'text/plain;charset=utf-8'
     });
-
```
Comment (tristen) [inline src/components/app.js]: Mind dropping this deletion?

File: src/components/app.js [hunk 2]
```diff
@@ -120,6 +132,7 @@ let App = class App extends React.PureComponent {
     const coordinates = this.calculateCoords(empty.querySelector('svg'));
     const paths = empty.querySelectorAll('path');
 
+
```
Comment (tristen) [inline src/components/app.js]: And this ws addition?

File: src/web-workers/svgo.js [hunk 3]
```diff
@@ -60,7 +60,7 @@ const pluginsData = {
   inlineStyles,
   minifyStyles,
   convertStyleToAttrs,
-  cleanupIDs,
+  // cleanupIDs,
   //prefixIds,
```
Comment (tristen) [inline src/web-workers/svgo.js:67]: 🤔 not seeing the reference to this variable. and can we add a quick note why they are commented out?

File: src/path-to-coordinates.js [hunk 4]
```diff
@@ -3,15 +3,20 @@ import { range } from 'lodash';
...
   // Always include the max value in the range.
   // This is helpful for detecting closed polygons vs lines
   getRange.push(numPoints);
 
-  return getRange.map(function(i) {
-    const point = path.getPointAtLength(length * i / numPoints);
-    return [point.x * scale + translateX, point.y * scale + translateY];
-  });
+  return {
+    id,
```
Comment (tristen) [inline src/path-to-coordinates.js:12]: Can we just pass `path` here? 

```js
return {
  ...path,
  coords: [ /*... */ ]
}
```

**Review by tristen — APPROVED** [review]
Just left a couple ideas! We should also use the squash button for merging this in.

File: src/path-to-coordinates.js (hunk 4 above)
Comment (lobenichou) [reply src/path-to-coordinates.js:12]: You want to pass the whole `path` here? Why? For now, we only need the id. If we want to handle more attribute in the future, we can but right now we would be passing something like this: `<path id="SC" d="M594.979 316.383l8.844 5.106 8.845-5.106 8.844 5.106 8.845-5.106 8.844 5.106v10.213l-8.844 5.107v10.212l-8.845 5.107v10.212l-8.844 5.107-8.845-5.107-8.844 5.107-8.845-5.107v-10.212l-8.844-5.107v-10.212l-8.845-5.107v-10.213l8.845-5.106 8.844 5.106z"></path>`

Comment (tristen) [reply src/path-to-coordinates.js:12]: Oh I see. I thought this was passing a regular object. Yeah, I was thinking if we wanted to include more in the future we wouldn't need to adjust this code each time. It also saved from [the](https://github.com/mapbox/svg-to-geojson/blob/71ad9131dea3ef3879caf09d9edbc195848f722a/src/components/app.js#L76-L78) [two](https://github.com/mapbox/svg-to-geojson/blob/94380336794d8baec68262576127c0b2cb5c218f/src/path-to-coordinates.js#L6) conditional checks you have here.

Comment (lobenichou) [reply src/path-to-coordinates.js:12]: Can I resolve for now?

**Pull request merged**